pkg/nvidia/Dockerfile

```diff
@@ -0,0 +1,105 @@
+# syntax=docker/dockerfile-upstream:1.5.0-rc2-labs
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+FROM lfedge/eve-alpine:82df60e43ab9f8c935584b8c7b4d0a4b0271d608 as build-base
+ENV BUILD_PKGS="autoconf automake build-base coreutils gettext git glib-dev go libtool libmd-dev ncurses-dev tar xz-dev yq zstd-dev"
+
+RUN eve-alpine-deploy.sh
+
+# This is an arm64 (only) package, nothing to do for other architectures
```

Comment:
Not critical, but is there a way to avoid running most of this file if we are not doing ARM build? Just the build packages installation takes ~10 seconds of time....

Reply:
Yeah, I can move the RUN eve-alpine-deploy.sh to the arm64 container....

Reply:
If it's a one-minute change, it would be nice to have it. If it's complicated and will require careful retesting, I can survive 10 extra seconds spent during the build.

Reply:
done.